Makefile

```diff
@@ -334,7 +341,7 @@ build/chart/index.yaml.$(YEAR_MONTH_DAY): build/chart/index.yaml
 
 build/chart/: build/chart/index.yaml build/chart/index.yaml.$(YEAR_MONTH_DAY)
 
-install-large-chart: build/toolchain/bin/helm$(EXE_EXTENSION)
+install-large-chart: delete-chart build/toolchain/bin/helm$(EXE_EXTENSION)
```

Comment:
This is destructive. If I want to apply an update to the chart I'll have to lose everything. Please remove this.